.github/workflows/build-app.yml

```diff
@@ -5,5 +5,9 @@ jobs:
   build-woof-target:
     runs-on: macos-13
     steps:
-      - uses: actions/checkout@v3
-      - run: ./scripts/build-ios-simulator.sh
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup Xcode 14.3
+        run: sudo xcode-select --switch /Applications/Xcode_14.3.app
```

Comment:
Please extract the command into a separate bash script (`set-up-environment.sh` or similar).

Reply:
Fixed with https://github.com/ios-course/ironfoudation-team-project/pull/188/commits/e65e3269a0ac9e61c73db68409846e8f4db93658